config/Migrations/20213028210100_add_whmcs_fields.php

```diff
@@ -0,0 +1,26 @@
+<?php
+use Migrations\AbstractMigration;
+
+class AddWhmcsFields extends AbstractMigration
+{
+
+    public $autoId = false;
+
+    public function change()
+    {
+        $table = $this->table('users');
+        $table
+            ->addColumn('whmcs_real_user_id', 'integer', [
```

Comment:
If there's already a `whmcs_user_id` column, why do we need this one? If the other one is no longer "real", why do we need it?

Reply:
WHMCS since the last update only passes the user id not the parent account and we need a way to associate them.